package.json

```diff
@@ -27,17 +27,21 @@
     "@babel/core": "^7.14.6",
     "@babel/plugin-proposal-class-properties": "^7.17.12",
     "@playwright/test": "1.25.1",
+    "@saucelabs/cucumber-reporter": "^0.2.0",
```

Comment:
See my comment for https://github.com/saucelabs/sauce-playwright-runner/pull/171#discussion_r986270060, does this work?

Reply:
This reporter works 🤔 I set the cucumber bin as this way.
```
const cucumberBin = path.join(process.cwd(), 'node_modules', '@cucumber', 'cucumber', 'bin', 'cucumber-js');
```

Reply:
Doesn't work for me: https://app.saucelabs.com/tests/77ab349bb01e463c856170d21ba7581d

Reply:
fixed in https://github.com/saucelabs/sauce-cucumber-reporter/pull/4/files